crates/core/src/repository.rs

```diff
@@ -286,7 +286,7 @@ impl Repository<NoProgressBars, ()> {
     /// * [`BackendAccessErrorKind::BackendLoadError`] - If the specified backend cannot be loaded, e.g. is not supported
     ///
     /// [`BackendAccessErrorKind::BackendLoadError`]: crate::error::BackendAccessErrorKind::BackendLoadError
-    pub fn new(opts: &RepositoryOptions, backends: RepositoryBackends) -> RusticResult<Self> {
+    pub fn new(opts: &RepositoryOptions, backends: &RepositoryBackends) -> RusticResult<Self> {
```

Comment:
Is this necessary? This breaks the API...

Reply:
It's not necessary, but we don't use it, so essentially the one before was unnecessary. I know it breaks the API, but maybe it's good to do that now, also the next release version is 0.3.0, so we essentially are already having API breaking changes, so that should be ok.